GliaWidgets/CallVisualizer/VisitorCode/VisitorCodeViewModel.swift

```diff
@@ -0,0 +1,78 @@
+import UIKit
+import SalemoveSDK
+
+extension CallVisualizer {
+    class VisitorCodeViewModel {
+        let presentation: CallVisualizer.Presentation
+        let visitorCodeAlertStyle: AlertStyle
+        var environment: Environment
+        var delegate: (Delegate) -> Void
+        var timer: FoundationBased.Timer?
+        var theme: Theme
+        var receivedVistorCode: VisitorCode? {
+            didSet {
+                notifyPropsUpdated()
+                scheduleVisitorCodeRefresh()
+            }
+        }
+
+        init(
+            presentation: CallVisualizer.Presentation,
+            visitorCodeAlertStyle: AlertStyle,
```

Comment:
could you clarify why you decided to pass alertStyle separately from theme?

Reply:
Because the context is Alert and the rest of the theme won't contribute laying out the UI

Reply:
is visitor code UI style out of theme?

Reply:
fixed [9880820](https://github.com/salemove/ios-sdk-widgets/pull/448/commits/9880820e952a63279fcac44568e7764c9be0f5a0)